controlplane/kubeadm/controllers/helpers.go

```diff
@@ -62,33 +61,32 @@ func (r *KubeadmControlPlaneReconciler) reconcileKubeconfig(ctx context.Context,
 			controllerOwnerRef,
 		)
 		if errors.Is(createErr, kubeconfig.ErrDependentCertificateNotFound) {
-			return errors.Wrapf(&capierrors.RequeueAfterError{RequeueAfter: dependentCertRequeueAfter},
-				"could not find secret %q, requeuing", secret.ClusterCA)
+			return ctrl.Result{RequeueAfter: dependentCertRequeueAfter}, nil
```

Comment:
Do we want to remove the error msg here? It may be useful in KCP line 261 when we log out the error. 

Also, the error should be `secret.Kubeconfig` instead of `secret.ClusterCA`.

Reply:
We are replacing a RequeueAfterError with a RequeueAfter, no need for the error anymore